tools/cas/cas.go

```diff
@@ -127,6 +125,10 @@ func main() {
 	if err := cachetools.GetBlobAsProto(ctx, bsClient, ind, msg); err != nil {
 		log.Fatal(err.Error())
 	}
-	out, _ := prototext.Marshal(msg)
+	printMessage(msg)
+}
+
+func printMessage(msg proto.Message) {
+	out, _ := prototext.MarshalOptions{Multiline: true}.Marshal(msg)
```

Comment:
Is this different from proto.MarshalTextString()?

Reply:
I think since we migrated from the github.com protobuf package to google.golang.org package, `proto.MarshalTextString()` is no longer available and prototext.Marshal is the new way to do it